lib/utils.ts

```diff
@@ -35,3 +36,21 @@ export const isValidNumber = (value: string) => {
 export function timeFromNow(date: Date | string) {
   return dayjs(date).fromNow();
 }
+
+export function findCollectibleUrlInCastEmbeds(embeds: string[]) {
```

Comment:
each file should do EXACTLY one thing
- actual: this file defines `multiple` functions
- expected: a new file is created solely for the definition of `findCollectibleUrlInCastEmbeds`